pom.xml

```diff
@@ -623,6 +620,12 @@
 						<include>**/*TestCase.java</include>
 						<include>**/*Tests.java</include>
 					</includes>
+					<!--
+						-Djava.net.preferIPv4Stack=true is set since embedded spaces take ages to start on some
+						developer machines without it. See also:
+						http://stackoverflow.com/questions/30997298/why-is-multicastsocket-setnetworkinterface-much-slower-using-java-8-runtime
+					-->
+					<argLine>-Djava.net.preferIPv4Stack=true</argLine>
```

Comment:
Did we conclude whether we need this or not?

Reply:
I added it because I couldn't get the project to work without it, and I think its still related to the problems we have discussed earlier with regards to `gs-test` and multicast. 

I will try without this again and see if it could work without it.